bin/pycbc_convertinjfiletohdf

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2021 Alex Nitz
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+""" This program converts standard injection file formats (e.g. XML format
+or the LIGO HDF format) into a PyCBC hdf format that includes a template hash
```

Comment:
Is this still true (ie there is a template hash in injection files ??)

Reply:
It never was true. I think this was copied from the template bank converter without updating. Fixed.